src/lightning_app/utilities/cluster_logs.py

```diff
@@ -43,59 +46,95 @@ def _push_log_events_to_read_queue_callback(read_queue: queue.PriorityQueue):
     Returns callback function used with `on_message_callback` of websocket.WebSocketApp.
     """
 
-    def callback(ws_app: WebSocketApp, msg: str):
-        # We strongly trust that the contract on API will hold atm :D
-        event_dict = json.loads(msg)
-        labels = _ClusterLogEventLabels(**event_dict["labels"])
-
-        if "message" in event_dict:
-            message = event_dict["message"]
-            timestamp = dateutil.parser.isoparse(event_dict["timestamp"])
-            event = _ClusterLogEvent(
-                message=message,
-                timestamp=timestamp,
-                labels=labels,
-            )
-            read_queue.put(event)
+    def callback(_: WebSocketApp, msg: str):
+        for ev in _parse_log_event(msg):
+            read_queue.put(ev)
 
     return callback
 
 
+def _parse_log_event(msg: str) -> List[_ClusterLogEvent]:
+    # We strongly trust that the contract on API will hold atm :D
+    event_dict = json.loads(msg)
+    labels = _ClusterLogEventLabels(**event_dict["labels"])
+    log_events = []
+
+    if "message" in event_dict:
+        message = event_dict["message"]
+        timestamp = dateutil.parser.isoparse(event_dict["timestamp"])
+        event = _ClusterLogEvent(
+            message=message,
+            timestamp=timestamp,
+            labels=labels,
+        )
+        log_events.append(event)
+    return log_events
+
+
 def _cluster_logs_reader(
-    client: LightningClient,
+    logs_api_client: _ClusterLogsSocketAPI,
     cluster_id: str,
-    start: arrow.Arrow,
```

Comment:
Let's remove arrow dep.

Reply:
This can be in follow up PR, this one is focused on fixes